data/ecosystem/vendors.yaml

```diff
@@ -41,6 +41,13 @@
   contact: 'hello@betterstack.com'
   oss: false
   commercial: true
+- name: Control Plane
+  distribution: false
+  nativeOTLP: true
+  url: 'https://controlplane.com'
```

Comment:
please point to the documentation that indiciates that your product supports OTLP

Reply:
fixed